src/client.toit

```diff
@@ -219,13 +226,19 @@ class Client:
   run_:
     while not task.is_canceled:
       remaining_keep_alive_us := keep_alive_.in_us - (Time.monotonic_us - last_sent_us_)
-      packet := ?
+      packet := null
       if remaining_keep_alive_us <= 0:
         packet = null
       else:
         remaining_keep_alive := Duration --us=remaining_keep_alive_us
         // Timeout returns a `null` packet.
-        packet = transport_.receive --timeout=remaining_keep_alive
+        exception := catch:
```

Comment:
Maybe you should be using `--trace=(: should_trace_exception it)` here?

Reply:
done.